root/static/scripts/edit/externalLinks.js

```diff
@@ -1447,6 +1453,10 @@ function isGoogleAmp(url) {
   return /^https?:\/\/([^/]+\.)?google\.[^/]+\/amp/.test(url);
 }
 
+function isExample(url) {
+  return /^https?:\/\/(?:[^/]+\.)?example\.\w+(?:\/.*)$/.test(url);
```

Comment:
Keep in mind that *technically* only .com, .org, .edu and .net are actually official example URLs - example.es or example.fr could be legit addresses.

Reply:
Right but they seem to be currently parked by ads, no chance it could be legit any time soon. ;)